templates/airtable/.nango/schema.ts

```
@@ -0,0 +1,86 @@
+// ---------------------------
+// This file was generated by Nango (v0.61.2)
+// You can version this file
+// ---------------------------
+
+export interface SyncMetadata_airtable_tables {
+};
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)